repository/stamp.go

```diff
@@ -93,4 +99,10 @@ type StampRepository interface {
 	// 存在しないスタンプがあった場合、ArgumentErrorを返します。
 	// DBによるエラーを返すことがあります。
 	ExistStamps(stampIDs []uuid.UUID) (err error)
+	// 成功した場合、(統計情報, nil)を返します。
```

Comment:
忘れてましたが、Goのコメントは最初にそのメソッド/関数/変数の名前を書くことが多いです。（上のメソッドを見てもらえれば`CreateStamp`とかで始まっています）
↓にしてください :pray:
```suggestion
	// GetStampStats 成功した場合、(統計情報, nil)を返します。
```